verification/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -163,6 +163,13 @@ def run_build(litani, jobs):
     cmd = [str(litani), "run-build"]
     if jobs:
         cmd.extend(["-j", str(jobs)])
+    # Restrict concurrency during CI to avoid 'out of memory' errors
+    # when proofs that require a lot of memory run concurrently
+    else:
+        factor   = 0.75
```

Comment:
I suggest to make this an argument to the script with a default value of 0.75. Then there's no need for a comment at all, instead write a help text that users will see.

Reply:
I have added this argument as "parallel factor" as suggested. I would like to leave the comment since there is no reason to do that in a normal situation.